packages/site/src/components/ComponentLinks.tsx

```diff
@@ -9,130 +9,87 @@ import { useAtlantisSite } from "../providers/AtlantisSiteProvider";
  * @returns ReactNode
  */
 export const ComponentLinks = ({
+  key,
   links,
   goToProps,
   goToUsage,
   goToDesign,
   webEnabled,
   mobileEnabled,
 }: {
+  readonly key: string;
   readonly links?: ContentExportLinks[];
   readonly goToProps: (type: string) => void;
   readonly goToUsage: (type: string) => void;
   readonly goToDesign: () => void;
   readonly webEnabled: boolean;
   readonly mobileEnabled: boolean;
 }) => {
-  const [hlinks, setHlinks] = useState<Element[] | null>(null);
-
-  useEffect(() => {
-    const hdd = document.querySelectorAll("[data-heading-link]");
-
-    if (hdd.length > 0) {
-      setHlinks(Array.from(hdd));
-    }
-  }, []);
-
-  const click = (e: MouseEvent) => {
-    e.preventDefault();
-    const id = e.currentTarget?.getAttribute("href")?.replace("#", "");
-
-    if (id) {
-      goToDesign();
-      setTimeout(() => {
-        const element = document.getElementById(id);
-
-        if (element) {
-          element.scrollIntoView({ behavior: "smooth" });
-        }
-      }, 100);
-    }
-  };
   const { isMinimal } = useAtlantisSite();
   if (isMinimal) return null;
 
   return (
-    <aside
-      style={{
-        width: "200px",
-        padding: "36px var(--space-base)",
-        display: "flex",
-        flexDirection: "column",
-        flexShrink: "0",
-        boxSizing: "border-box",
-      }}
-    >
-      <Content spacing={"larger"}>
+    <Content spacing={"larger"}>
```

Comment:
The diff looks much more involved than what was actually changed. This was essentially just pulling the anchor link functionality out into its own component